inc/template-tags.php

```diff
@@ -116,51 +116,72 @@ function primer_the_page_title() {
 /**
  * Display navigation to next/previous set of posts when applicable.
  *
- * @global WP_Query $wp_query
- * @since  1.0.0
+ * @deprecated NEXT No longer supported as the pagination funciton. See primer_pagination().
```

Comment:
@EvanHerman I'm thinking we should create `inc/deprecated.php` to house all the cruft.